src/commands/context/list.ts

```diff
@@ -0,0 +1,47 @@
+/*********************************************************************
+ * Copyright (c) 2020 Red Hat, Inc.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ **********************************************************************/
+
+import { Command } from '@oclif/command'
+import { cli } from 'cli-ux'
+
+import { CheServerLoginManager } from '../../api/che-login-manager'
+
+export default class List extends Command {
+  static description = 'shows all existing login sessions'
+
+  async run() {
+    const loginManager = await CheServerLoginManager.getInstance(this.config.configDir)
+    const logins = loginManager.getAllLogins()
```

Comment:
If we are talking about context, then it is needed to rename all functions correspondingly.

Reply:
Disagree. In code I have login manager, but the command sets context, i.e. login which should be used by default.